### PR TITLE
Jl/lists

### PR DESCRIPTION
**Pull request by jbieniosek**
The numbered lists don't display correctly in Learn Preview, not sure if they'll work in actual Learn... :(

### CODE REVIEW

**Comment by jbieniosek** [comment]
Fixed numbered list markdown issues

File: 01-intro-to-python/04-lists.md [hunk 1]
```diff
@@ -71,116 +69,156 @@ for i in new_list_2:
...
+    </details>
+
+1. The last piece of adding our new random word is replacing the constant `SNOWMAN_WORD` in the conditional test inside of the `snowman` function with the new `snowman_word` variable.
+
+    ```python
+
+    def snowman():
+        r = RandomWord()
+        snowman_word = r.word(word_min_length=SNOWMAN_MIN_WORD_LENGTH, word_max_length=SNOWMAN_MAX_WORD_LENGTH)
```
Comment (CheezItMan) [inline 01-intro-to-python/04-lists.md:108]: Just to fit on the screen

```suggestion
        snowman_word = r.word(
          word_min_length=SNOWMAN_MIN_WORD_LENGTH, 
          word_max_length=SNOWMAN_MAX_WORD_LENGTH)
```

File: 01-intro-to-python/04-lists.md [hunk 2]
```diff
@@ -71,116 +69,156 @@ for i in new_list_2:
...
+
+    # ... rest of file
+
+    ```
+1. Next, add the constants `SNOWMAN_MAX_WORD_LENGTH = 8` and `SNOWMAN_MIN_WORD_LENGTH = 5` with the other constants at the top of the file.    
+1. Last, add the following lines of code to the top of the `snowman` function:
+    ```python
+
+        r = RandomWord()
+        snowman_word = r.word(word_min_length=SNOWMAN_MIN_WORD_LENGTH, word_max_length=SNOWMAN_MAX_WORD_LENGTH)
```
Comment (CheezItMan) [inline 01-intro-to-python/04-lists.md:94]: Just to fit on the screen

```suggestion
        snowman_word = r.word(
          word_min_length=SNOWMAN_MIN_WORD_LENGTH, 
          word_max_length=SNOWMAN_MAX_WORD_LENGTH)
```

File: 01-intro-to-python/04-lists.md [hunk 3]
```diff
@@ -71,116 +69,156 @@ for i in new_list_2:
...
+
+    ```
+    <details>
+    <summary>Curious about these lines of code?  Click here for more!</summary>
+    The first line `r = RandomWord()` gives us a RandomWord object to work with.  We will not cover classes and objects in the pre-course material, but they will be a topic we will cover at Ada.  The next line, `snowman_word = r.word(word_min_length=SNOWMAN_MIN_WORD_LENGTH, word_max_length=SNOWMAN_MAX_WORD_LENGTH)` will call the function `word` on the the RandomWord object `r` and give us a random word.  We are passing two arguments (word_min_length and word_max_length) using keyword arguments.  Again, we will not be covering these topics further in the pre-course, but they will come up later in the Ada curriculum.  The arguments that we are passing to the function `word` will instruct `word` to give us an English word where the length is between the SNOWMAN_MIN_WORD_LENGTH and SNOWMAN_MAX_WORD_LENGTH.  Feel free to experiment with setting different values for the constants.
```
Comment (CheezItMan) [inline 01-intro-to-python/04-lists.md:99]:
```suggestion
    The first line `r = RandomWord()` gives us a `RandomWord` object to work with.  We will not cover classes and objects in the pre-course material, but they will be a topic we will cover at Ada.  
    
    The next line, `snowman_word = r.word(word_min_length=SNOWMAN_MIN_WORD_LENGTH, word_max_length=SNOWMAN_MAX_WORD_LENGTH)` will call the function `word` on the the RandomWord object `r` and give us a random word.  
    
 We are passing two arguments (word_min_length and word_max_length) using keyword arguments.  Again, we will not be covering these topics further in the pre-course, but they will come up later in the Ada curriculum.  The arguments that we are passing to the function `word` will instruct `word` to give us an English word where the length is between the SNOWMAN_MIN_WORD_LENGTH and SNOWMAN_MAX_WORD_LENGTH.  Feel free to experiment with setting different values for the constants.
```

File: 01-intro-to-python/04-lists.md [hunk 4]
```diff
@@ -71,116 +69,156 @@ for i in new_list_2:
...
-            print("You guessed a letter that's in the word!")
-        else:
-            print(f"The letter {user_input} is not in the word")
-            wrong_guesses_list.append(user_input)
-# ...
+    # ...
+            if user_input in snowman_word:
+                print("You guessed a letter that's in the word!")
+            else:
+                print(f"The letter {user_input} is not in the word")
```
Comment (CheezItMan) [inline 01-intro-to-python/04-lists.md:152]: Just trying to get things to fit in learn.

```suggestion
                print("You guessed a letter that's \
in the word!")
            else:
                print(f"The letter {user_input} is not in \
the word")
```

File: 01-intro-to-python/04-lists.md [hunk 5]
```diff
@@ -71,116 +69,156 @@ for i in new_list_2:
...
-            valid_input = True
-
-    return user_input_string
-
-```
+    def snowman():
+        # ...
+        wrong_guesses_list = []
+        while len(wrong_guesses_list) < SNOWMAN_WRONG_GUESSES:
+            user_input = get_letter_from_user(wrong_guesses_list)
```
Comment (CheezItMan) [inline 01-intro-to-python/04-lists.md:193]: Just a note, we need to rethink such long variable names like `wrong_guesses_list`.  Instead maybe name such things `wrong_guesses`, we don't need `_list`.

**Review by CheezItMan — COMMENTED** [review]
Overall pretty good, I made a few suggestions

**Pull request merged**